### PR TITLE
Add option to build ARM64 on Linux

### PR DESCRIPTION
**Pull request by slide**
This adds an option to build for ARM64 (aarch64) on Linux.

### CODE REVIEW

**Comment by slide** [comment]
Related to https://github.com/PowerShell/PowerShell/issues/7330

File: build.psm1 [hunk 1]
```diff
@@ -621,6 +630,7 @@ function PlaceUnixBinaries
 
     $RuntimePathLinux = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-x64/native') -Force
     $RuntimePathLinuxARM = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-arm/native') -Force
+    $RuntimePathLinuxARM64 = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-arm64/native') -Force
```
Comment (adityapatwardhan) [inline build.psm1]: Add a parameter to the function for passing in `$BinFolderLinuxARM64`. Then use `$RuntimePathLinuxARM64` like in line 640 for `Copy-Item`

Comment (adityapatwardhan) [reply build.psm1]: Also, `Start-BuildPowerShellNativePackage` function needs a parameter for linux-arm64.

File: .vsts-ci/windows.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ phases:
...
+      Start-Process -FilePath 'vs_BuildTools.exe' -ArgumentList '--quiet', '--norestart', '--locale en-US', '--add Microsoft.VisualStudio.Component.VC.Tools.ARM', '--add Microsoft.VisualStudio.Component.VC.Tools.ARM64', '--includeRecommended', '--add Microsoft.VisualStudio.Workload.VCTools', '--add Microsoft.VisualStudio.Component.Windows10SDK.16299.Desktop.arm', '--add Microsoft.VisualStudio.Component.VC.ATL.Spectre', '--add Microsoft.VisualStudio.Component.VC.ATLMFC.Spectre', '--add Microsoft.VisualStudio.Component.VC.ATL.ARM.Spectre', '--add Microsoft.VisualStudio.Component.VC.ATL.ARM64.Spectre', '--add Microsoft.VisualStudio.Component.VC.Runtimes.ARM.Spectre', '--add Microsoft.VisualStudio.Component.VC.Runtimes.ARM64.Spectre', '--add Microsoft.VisualStudio.Component.VC.Runtimes.x86.x64.Spectre' -Wait
```
Comment (adityapatwardhan) [inline .vsts-ci/windows.yml]: This change is unrelated. Please make this a separate PR.

File: .vsts-ci/linux.yml [hunk 3]
```diff
@@ -18,6 +18,8 @@ phases:
     matrix:
       Linux ARM Native:
         buildName: ubuntu.16.04
+      Linux ARM64 Native:
+        buildName: ubuntu.16.04
```
Comment (adityapatwardhan) [inline .vsts-ci/linux.yml]: Use the name defined in build.json instead of `ubuntu.16.04`. I believe this will be building arm32 again.

File: .vsts-ci/windows.yml (hunk 2 above)
Comment (SteveL-MSFT) [reply .vsts-ci/windows.yml]: The Windows build was failing without this change

**Comment by adityapatwardhan** [comment]
@slide Thank you for your contribution!
